app/annotationsList/annotationList.html

```diff
@@ -23,7 +23,7 @@
       </pane>
     </dropdown-toggle>
 
-    <span class="pagination-text"><strong>{{totalItems}}</strong> annotations</strong></span>
+    <span class="pagination-text"><strong>{{totalItems | number}}</strong> annotations</strong></span>
```

Comment:
This is not a variable but an angular filter, see here https://docs.angularjs.org/api/ng/filter/number